### PR TITLE
feat: Add operators data periodic fetcher

### LINKED ISSUE #1137: feat(telemetry): track operators events
# Description

Telemetry api should track the following events from Ethereum

- RegisteredOperator
- Stake updates
- Metadata updates

### PR DESCRIPTION
**Pull request by JulianVentura**
### Description

This PR adds a periodic data fetch of operands data making use of a `Task`.

### How to test

1. Run telemetry service without any operator registered
2. Go to `localhost:4000` and data should be empty
3. Register an operator
4. Go to `localhost:4000` and data from operator should have appeared
5. Run the registered operator
6. Go to `localhost:4000` and the operator's version should appear.

Closes #1137

### CODE REVIEW

**Review by avilagaston9 — APPROVED** [review]
LGTM

File: telemetry_api/lib/telemetry_api/periodic/operator_fetcher.ex [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  defp poll_serivce() do
+    receive do
+    after
+      @wait_time_ms ->
+        case Operators.fetch_all_operators() do
+          {:ok, _} -> :ok
+          {:error, message} -> IO.inspect "Couldn't fetch operators: #{IO.inspect message}"
+        end
+        poll_serivce()
```
Comment (Oppen) [inline telemetry_api/lib/telemetry_api/periodic/operator_fetcher.ex]:
```suggestion
  defp poll_service() do
    receive do
    after
      @wait_time_ms ->
        case Operators.fetch_all_operators() do
          {:ok, _} -> :ok
          {:error, message} -> IO.inspect "Couldn't fetch operators: #{IO.inspect message}"
        end
        poll_service()
```

File: telemetry_api/lib/telemetry_api/periodic/operator_fetcher.ex [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  alias TelemetryApi.Operators
+
+  wait_time_str = System.get_env("OPERATOR_FETCHER_WAIT_TIME_MS") ||
+    raise """
+    environment variable OPERATOR_FETCHER_WAIT_TIME_MS is missing.
+    """
+
+  @wait_time_ms (
+    case Integer.parse(wait_time_str) do
+      :error -> raise("OPERATOR_FETCHER_WAIT_TIME_MS is not a number")
```
Comment (Oppen) [inline telemetry_api/lib/telemetry_api/periodic/operator_fetcher.ex]: Might be a good idea to include the actual value.

File: telemetry_api/lib/telemetry_api/periodic/operator_fetcher.ex (hunk 1 above)
Comment (JulianVentura) [reply telemetry_api/lib/telemetry_api/periodic/operator_fetcher.ex]: Good catch!

File: telemetry_api/lib/telemetry_api/periodic/operator_fetcher.ex (hunk 2 above)
Comment (JulianVentura) [reply telemetry_api/lib/telemetry_api/periodic/operator_fetcher.ex]: Yep, that would help

**Pull request merged**